docs/zh/deploy/install_deploy.md

```diff
@@ -85,6 +85,10 @@ Debug build (NDEBUG not #defined)
 
 如果产生coredump，很可能时指令集不兼容问题，需要通过源码编译openmldb。源码编译参考[这里](./compile.md), 需要采用方式三完整源代码编译。
 
+## 守护进程启动方式
+
```

Comment:
update en docs

Reply:
en version of install_deploy is so far behind , ref https://github.com/4paradigm/OpenMLDB/issues/2694